templates/msgs/faq_list.haml

```diff
@@ -32,18 +32,17 @@
               - trans "Translations"
 
         %tbody
-          
-            %tr{ ng-repeat:"faq in faqs | filter:filterParents" }
-              %td
-                %a{ ng-href:"/faq/read/[[ faq.id ]]/" }
-                  [[ faq.question ]]
-              %td.col-xs-6
-                [[ faq.answer ]]
-              %td
-                [[ faq.language.name ]]
-              %td.text-center
-                %span.badge
-                  [[ faq.count ]]
+          %tr{ ng-repeat:"faq in faqs | filter:filterParents" }
```

Comment:
This seems a bit inefficient but I've left it as is for now. Would be better to only fetch non-translation FAQ instances on the server